<launch>

    <node name="key_driver" pkg="keyboard_driver" type="keyboard_driver" output="screen" respawn = "true" >
        <param name="freq" value="70" />
        <param name="x_vel" value="0.5" />
        <param name="z_ang" value="0.5"/>

        <remap from="pub_velocity_cmd" to="/m_robot/velocity_cmd"/>

    </node>

</launch>